src/Donations/Endpoints/ListDonations.php

```diff
@@ -126,6 +127,16 @@ public function registerRoute()
                             'columns',
                         ],
                     ],
+                    'postStatus' => [
```

Comment:
The post status is something we use under the hood for storage - however, the donation should just refer to this as _status_

```suggestion
                    'status' => [
```